README.md

```diff
@@ -60,41 +50,10 @@ to always keep macOS on and available.
 | `node['macos']['network_time_server']`  | `'time.windows.com'`    |
 | `node['macos']['time_zone']`            | `'America/Los_Angeles'` |
 
-### Xcode
-
-Installs the latest Xcode the platform supports. See the [Xcode resource documentation](https://github.com/Microsoft/macos-cookbook/blob/master/documentation/resource_xcode.md) if you need
-more flexibility.
-
-:warning: Requires a `credentials` data bag containing an `apple_id` data bag item,
-or a user/password pair set under `node['macos']['apple_id']`.
-
-**Usage:** `include_recipe 'macos::xcode'`
-
-| Attributes used                                        | Default value |
-|--------------------------------------------------------|---------------|
-| `node['macos']['xcode']['version']`                    | `'9.3'`       |
-| `node['macos']['xcode']['simulator']['major_version']` | `nil`         |
-| `node['macos']['apple_id']['user']`                    | `nil`         |
-| `node['macos']['apple_id']['password']`                | `nil`         |
-
-## Data Bags
-
-The `macos::xcode` recipe can utilize a `credentials` data bag with an `apple_id`
-data bag item. The item should contain valid Apple ID credentials. For example:
-
-**Example:**
-
-```json
-{
-  "id": "apple_id",
-  "apple_id": "farva@spurbury.gov",
-  "password": "0k@yN0cR34m"
-}
-```
-
```

Comment:
Are we sure we want to remove the entire **Data Bags** section of the README when `macos::xcode` [still supports this functionality](https://github.com/Microsoft/macos-cookbook/blob/release/3.0/resources/xcode.rb#L18-L22)?

Reply:
Also, if we are going to remove it, a [broken link](https://github.com/Microsoft/macos-cookbook/pull/193/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11) still exists at the top of the README.

Reply:
Nice catch - updating the resource doc with the content changes

Reply:
Fixed.